oeps/processes/oep-0054-core-contributors.rst

```diff
@@ -0,0 +1,249 @@
+      contribution - if the candidate is part of an organization, this should be
+      covered as part of the organization's signature to the Declaration of
+      Commitment to the Core Contributor Program, as covered in the `program
+      administration`_ section.
+   
+   #. Finally, the post must mention the date when the comment period ends,
+      which is at least 2 weeks from the start of the thread. Please be mindful
+      of times such as holiday breaks where people may have limited
+      availability.
+
+#. The sponsor or organic candidate posts a link to the Discourse post in the
+   ``#core-contributors`` Slack room, as well as any other Slack rooms relevant
+   to the CC role being discussed (for example, repo-specific rooms for coding
+   contributors, translation team rooms, etc). **The messages must indicate the
+   dates of the comment period.**
+
+#. During the comment period, other CCs should weigh in on the thread. As much
+   as possible, posts should conclude with a definitive "yes" or "no" vote.
+
+   #. If anyone has concerns they feel cannot be raised publicly, they should
+      consult with the Program Administrators to determine how to proceed.
+      Program administrators can be contacted at
+      ``cc-program-admins@tcril.org``. The Administrators should discuss the
+      concerns and try to figure out a way the person could post those concerns
+      themselves; if they cannot, the Administrators should post an anonymized
+      version of the concerns on the Discourse thread.
+
+   #. For those leaving a "no" vote: Remember to leave constructive criticism! A
+      "no" vote should be interpreted as "candidate isn't ready yet". In this
+      case, provide some tips as to how they could make it into the program. And
+      please err on the side of leaving that feedback, rather than not replying
+      at all. A good way to deliver this feedback is "Thank you for contributing
+      to the Open edX community. However, I feel you are not quite ready for
+      this role, for <reasons>. Some ways you could address these are <action
+      items>"
+
+#. Over the course of the comment period, the sponsor, sponsored candidate, or
+   organic candidate should respond to all comments that come in. It may also be
+   necessary to periodically nudge other CCs to respond to the thread.
+
+   #. At any time, the candidate may choose to withdraw from the process. The
+      candidate should leave a note on the thread, and close it if they wish.
+
+#. At the end of the comment period, a candidate is approved if there are at
+   least 5 (five) affirmative "yes" votes, and zero "no" votes.
+
+   #. For candidates coming from an organization, at least one affirmative vote
+      must come from a CC outside of the candidate's org. This is to ensure that
+      core contributors are evaluated on their ability to communicate with the
+      community, outside of their org.
```

Comment:
I added this after conversation within tCRIL engineering. We think it's important a new CC has the approval of at least one person outside their org. Being able to communicate within your own org/team is different than being able to reach out to and communicate with someone cross-organizationally.